dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/RumFeature.kt

```diff
@@ -86,7 +89,10 @@ internal object RumFeature : SdkFeature<Any, Configuration.Feature.RUM>() {
         configuration.userActionTrackingStrategy?.let { actionTrackingStrategy = it }
         configuration.longTaskTrackingStrategy?.let { longTaskTrackingStrategy = it }
 
-        initializeVitalMonitors()
+        if (configuration.vitalsMonitorUpdateFrequency != VitalsUpdateFrequency.NEVER) {
+            initializeVitalMonitors(configuration.vitalsMonitorUpdateFrequency.frequencyInMs)
+        }
```

Comment:
Two quick things, first of all, it'd make the `onInitialize` method easier to read if you'd call just 

`initializeVitalMonitors(configuration.vitalsMonitorUpdateFrequency)` 

and did the check just there. Also let's only extract the `periodInMs` when it's actually needed

Reply:
I had it like that in the first place, honestly was not sure which way would be better as doing the check in the method means a premature return, sometimes I like to avoid this. But yeah, I think it looks cleaner like that.